### DISCUSSION TITLE
SBAS Network with gaps

### DISCUSSION THREAD

**Discussion by pbrotoisworo** [insarlab/MintPy#966] [Q&A] [answered]
What is the recommended approach if data coverage has gaps? Assuming we have a network like something below what are the effects of gaps in the network? Should we process the data separately instead of one big timeseries? The area I'm looking at does not have consistent Sentinel-1 coverage.

Please disregard the low coherence, I'm just running some tests with the data and subsets.
Thanks in advance.

![image](https://user-images.githubusercontent.com/32087633/221422006-50a6c80e-4d49-49e4-b15d-7bd8ffbcca21.png)

**Comment by yunjunz** [comment, marked as answer]
The gaps in the network will introduce bias in the estimated time-series, as the design matrix does not have full rank anymore, the phase velocity objective function (as introduced in Berardino et al., 2002) helps with this situation, but could not eliminate it.

Process each subset separately works if the time coverage still suits your purpose. 

I want to ask: why there are gaps in the first place? Although the temporal baseline there is large, we could still generate interferograms to connect them, such as in isce2/topsStack, the nearest 3 connection network will give a fully connected network, regardless of 36-day or 108-day.

**Reply by pbrotoisworo** [reply]
There are some gaps because I wanted to maximize the coherence and minimize noise. The SBAS network used Sentinel-1 image pairs with temporal baseline of 12, 24, 36 days. The specific area I was looking at had some gaps in the data because my search filters in the ASF site also looked for specific frame to maximize overlap when using cutGeotiffs.py from hyp3lib tools.

Thanks for your advice and insight. I can try to reprocess this data to get better results.
